catroid/src/main/java/org/catrobat/catroid/formulaeditor/SensorHandler.java

```diff
@@ -136,6 +138,13 @@ public static void startSensorListener(Context context) {
 		instance.registerListener(instance);
 
 		instance.sensorManager.registerListener(instance, Sensors.LOUDNESS);
+		instance.sensorManager.registerListener(instance, Sensors.DATE_YEAR);
+		instance.sensorManager.registerListener(instance, Sensors.DATE_MONTH);
+		instance.sensorManager.registerListener(instance, Sensors.DATE_DAY);
+		instance.sensorManager.registerListener(instance, Sensors.DATE_WEEKDAY);
+		instance.sensorManager.registerListener(instance, Sensors.TIME_HOUR);
+		instance.sensorManager.registerListener(instance, Sensors.TIME_MINUTE);
+		instance.sensorManager.registerListener(instance, Sensors.TIME_SECOND);
```

Comment:
I'm not sure if you really need to register any listeners, but please correct me if I'm wrong.